Fixed app-routing

lmtataru opened the pull request:
Fixed app routing by creating the home-page route and set the wildcards order.

lmtataru committed:
Fixed app-routing

Fixed app routing by creating the home-page and set the wildcards order.

jakePetrulla reviewed (changes requested):
when using the wildcard **, use a redirect to the 404 page instead of loading the component. We need to be able to route to 404 in some cases so it's better to also have a 404 route.

https://www.techiediaries.com/angular-9-8-path-redirection-and-handling-404-using-wildcard-routes/

lmtataru committed:
Update app-routing - page not found

Created a 404 path and then redirect the wildcard to the 404 path.
Also deleted the navbar view from app-component.html.
